### ISSUE TITLE
UnknownIssuer error with private CA

### ISSUE THREAD

**Issue by austinbutler** [ducaale/xh#224]
With a private CA chain added to `/etc/pki/tls/certs/ca-bundle.crt` I get `error trying to connect: invalid certificate: UnknownIssuer`. The same request works with `curl` or `httpie`. If I use the `--native-tls` flag with `xh` it works.

Without being all that familiar with `rustls`, two potential suggestions:
1. Better documentation around how SSL certs are found/used?
2. Integrate [rustls-native-certs](https://github.com/rustls/rustls-native-certs) so that system certificate store just works?

**Comment by ducaale** [comment]
By default, the reqwest library uses `webpki-roots` but can also integrate with `rustls-native-certs` via the `rustls-native-certs` feature.

@austinbutler can you try compiling `xh` after [adding both](https://github.com/seanmonstar/reqwest/pull/1058#issuecomment-710417994) `rustls-tls-webpki-roots` and `rustls-tls-native-roots` to reqwests's enabled features? thanks

```diff
[dependencies.reqwest]
version = "0.11.6"
default-features = false
- features = ["rustls-tls", "json", "gzip", "brotli", "deflate", "multipart", "blocking", "socks", "cookies"]
+ features = ["rustls-tls", "rustls-tls-webpki-roots", "rustls-tls-native-roots", "json", "gzip", "brotli", "deflate", "multipart", "blocking", "socks", "cookies"]
```

**Comment by austinbutler** [comment]
Yep that seems to do it. Should I open a PR?

**Comment by ducaale** [comment]
Yes, that would be very much appreciated!

**Comment by ducaale** [comment]
This is now fixed in xh v0.15.0
